src/objects.rs

```diff
@@ -1291,12 +1291,31 @@ pub struct Geometry {
     pub geometry: GeoGeometry<f64>,
 }
 
+impl Default for Geometry {
+    fn default() -> Geometry {
+        use geo_types::Point;
+        let point: Point<f64> = (0.0, 0.0).into();
+        Geometry {
+            id: "default_geometry".to_string(),
+            geometry: point.into(),
+        }
+    }
+}
```

Comment:
I'm not found of that, is it really needed?

Reply:
fond :stuck_out_tongue_winking_eye:

Reply:
not fan too
it is used by WithId trait for Geometry

```rust
impl WithId for Geometry {
    fn with_id(id: &str) -> Self {
        let mut r = Self::default();
        r.id = id.to_owned();
        r
    }
}
```